tests/chainer_tests/functions_tests/math_tests/test_matmul.py

```diff
@@ -10,24 +9,44 @@
 from chainer import testing
 from chainer.testing import attr
 from chainer.testing import condition
+from chainer.utils import type_check
 
 
 class _TestMatMul(unittest.TestCase):
 
```

Comment:
`_TestMatMul` is no longer needed, because it's subclassed only by `TestMatMul`. How about merging them?